modules/pagerduty-service-integration-email/generic-email-inbound-integration.tf

```diff
@@ -0,0 +1,21 @@
+resource "pagerduty_service_integration" "generic_email_inbound_integration" {
+  name              = var.name
+  service           = var.service_id
+  integration_email = var.integration_email
+  # hardcoded now since we don't support multiple email filters.
```

Comment:
not clear what this means

Reply:
the underlying resource has support for taking in an array of email filter objects and having AND/OR between them. IMO This comment will make sense to those that look at the doc for this resource and wonder why we didn't build support for multiple email filters.